mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -36,148 +40,140 @@ interface PasscodePromptProps {
 }
 
 const PromptContent = React.memo((props: PasscodePromptProps): JSX.Element => {
+  const { isLight } = useThemeContext()
   return (
     <>
-      <ThemedTouchableOpacity
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
+      <ThemedTouchableOpacityV2
+        light={tailwind('bg-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-100')}
         onPress={() => props.onCancel(USER_CANCELED)}
-        style={tailwind('items-end pt-2 pr-2')}
+        style={[tailwind('items-end pt-6 pr-6'), { borderTopLeftRadius: 15, borderTopRightRadius: 15 }]}
```

Comment:
can use `rounded-t-2xl` class here, lets not use inline style as much as possible